lang/test/org/partiql/lang/syntax/SqlParserTestBase.kt

```diff
@@ -95,7 +114,7 @@ abstract class SqlParserTestBase : TestBase() {
         val expectedPigAst = PartiqlAst.build { expectedPigBuilder() }.toIonElement().toString()
 
         // Refer to comments inside the main body of the following function to see what checks are performed.
-        assertExpression(source, expectedPigAst)
+        assertExpression(source, expectedPigAst, true)
```

Comment:
This `true` shouldn't be necessary since the default value for `assertExpression`'s `roundTrip` argument is `true`.

Reply:
Not necessary, but I do like the explicitness to directly compare with `assertExpressionNoRoundTrip`